lib/diagnostic.rb

```diff
@@ -8,21 +8,21 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = ruby 'lib/example.rb'
```

Comment:
you left ruby out of the string, should be `'ruby lib/example.rb'` but close enough!